src/bot.py

```diff
@@ -255,71 +404,117 @@ async def update(ctx):
 async def twan(ctx):
-    await ctx.send(EasterEggs.Twan())
+    await ctx.reply(EasterEggs.Twan())
 
 
-@client.command(name='sad', aliases=[':('], pass_context=True)
+@client.command(name="sad", aliases=[":("], pass_context=True)
 async def sad(ctx):
-    await ctx.send(EasterEggs.Sad())
+    await ctx.reply(EasterEggs.Sad())
 
 
-@client.command(name='smh', aliases=['myhead'], pass_context=True)
+@client.command(name="smh", aliases=["myhead"], pass_context=True)
 async def smh(ctx):
-    await ctx.send(EasterEggs.Smh())
+    await ctx.reply(EasterEggs.Smh())
 
 
-@client.command(name='turhols', aliases=['<:IncognitoTurhol:540327644089155639>'], pass_context=True)
+@client.command(name="turhols", aliases=["<:IncognitoTurhol:540327644089155639>"], pass_context=True)
 async def turhols(ctx):
-    await ctx.send(EasterEggs.Turhols())
+    await ctx.reply(EasterEggs.Turhols())
 
 
-@client.command(name='pika', aliases=['<:pika:538182616965447706>'], pass_context=True)
+@client.command(name="pika", aliases=["<:pika:538182616965447706>"], pass_context=True)
 async def pika(ctx):
-    await ctx.send(EasterEggs.Pika())
+    await ctx.reply(EasterEggs.Pika())
 
 
-@client.command(name='zappa', aliases=['zapp', 'zac', '<:zappa:632813684678197268>', '<:zapp:632813709579911179>'], pass_context=True)  # noqa
+@client.command(name="zappa", aliases=["zapp", "zac", "<:zappa:632813684678197268>", "<:zapp:632813709579911179>"], pass_context=True)  # noqa
 async def zappa(ctx):
-    await ctx.send(EasterEggs.Zappa())
+    await ctx.reply(EasterEggs.Zappa())
 
 
-@client.command(name='duis', pass_context=True)
+@client.command(name="duis", pass_context=True)
 async def duis(ctx):
-    await ctx.send(EasterEggs.Duis())
+    await ctx.reply(EasterEggs.Duis())
+
+
+@client.command(name="furry", pass_context=True)
+async def furry(ctx):
+    await ctx.reply(EasterEggs.Furry())
+
+
+@client.command(name="don", pass_context=True)
+async def don(ctx):
+    await ctx.reply(EasterEggs.Don())
+
+
+@client.command(name="daffy", pass_context=True)
+async def daffy(ctx):
+    await ctx.reply(EasterEggs.Daffy())
+
+
+@client.command(name="giddy", pass_context=True)
+async def giddy(ctx):
+    await ctx.reply(EasterEggs.Giddy())
+
+
+@client.command(name="nodought", pass_context=True)
+async def nodought(ctx):
+    await ctx.reply(EasterEggs.NoDought())
 
 
-@client.command(name='normq', pass_context=True)
+@client.command(name="coolio", pass_context=True)
+async def coolio(ctx):
+    await ctx.reply(EasterEggs.Coolio())
+
+
+@client.command(name="normq", pass_context=True)
 async def normq(ctx):
-    messages: List[str or Embed] = EasterEggs.NormQ()
-    for msg in messages:
-        if (isinstance(msg, Embed)):
-            await ctx.send(embed=msg)
-        else:
-            await ctx.send(msg)
+    blueTeam, _ = Queue.getTeamList()
+    if (len(blueTeam) >= 1):
+        await ctx.send("Duis says I am not supposed to queue, but I don't listen to players worse than me...")
```

Comment:
These two messages are duplicated in both branches. Move those above the `if`.

Reply:
Resolved.